src/pages/Lend/LendByCategoryPage.vue

```diff
@@ -384,6 +385,44 @@ export default {
 				);
 			}
 		},
+		initializeRecommendedLoansRowExp() {
+			// experiment: CASH-1801 "Loans For You" Recommendation Row
+			// get assignment
+			let recommendationChannel;
+			const recommendedLoansRowEXP = this.apollo.readFragment({
+				id: 'Experiment:recommendation_channel',
+				fragment: experimentVersionFragment,
+			}) || {};
+			this.recommendedLoansRowExpVersion = recommendedLoansRowEXP.version;
+			// Logged in user and non-users are included in this experiment,
+			// logged-in users are automatically tracked with their id
+			// Fire Event for Exp CASH-1801
+			this.$kvTrackEvent(
+				'Lending',
+				'EXP-CASH-1801-Feb2020',
+				this.recommendedLoansRowExpVersion === 'shown' ? 'b' : 'a'
+			);
+			// Divide loggedIn users into control and variant for rec channel
+			if (recommendedLoansRowEXP.version === 'variant-a') {
```

Comment:
@go-bears You can remove both of these blocks. The kvTrackEvent is already getting fired above and we don't want to include console.logs code that might run in prod.

Reply:
Also. if you wanted to track the recommendation channel you could add a 4th param (value) to the kvTrackEvent with that value so long as it's a number 0 or 1 or 42 or whatever.

Reply:
thanks for the insight!